ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/monitor_layer1.schema.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  monitor_layer1:
+    type: dict
+    description: "Enable SYSLOG messages on transceiver SMBus communication failures."
+    keys:
+      enabled:
+        type: bool
+        required: true
+        description: "Enable monitor layer1"
```

Comment:
Will the config of the main stanza have any effect on EOS? I mean does it make sense to have this or should we just check for all three keys in the template to decide if we output anything?